### PR TITLE
dz60 Iso de 5x1u split right shift

### PR DESCRIPTION
**Pull request by 1r00t**
## Description

I added a German layout to the DZ60. It's a copy from iso_de_andys8
## Types of Changes
keymap.c
info.json
dz60.h

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by 1r00t** [comment]
why is this failing? It lists different keymaps from different people but not mine.

File: keyboards/dz60/dz60.h [hunk 1]
```diff
@@ -305,6 +305,33 @@
...
+ * │10   │12 │13 │14 │15 │16 │17 │18 │19 │1a │1b │1c │1d │     │
+ * ├─────┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┐2d  │
+ * │20    │22 │23 │24 │25 │26 │27 │28 │29 │2a │2b │2c │1e │    │
+ * ├────┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴───┴┬───┤
+ * │30  │31 │32 │33 │34 │35 │36 │37 │38 │39 │3a │3b │3d    │3e │
+ * ├────┼───┴┬──┴─┬─┴───┴───┴───┴───┴───┴──┬┴──┬┴──┬┴──┬───┼───┤
+ * │40  │41  │43  │46                      │4a │4b │4c │4d │4e │
+ * └────┴────┴────┴────────────────────────┴───┴───┴───┴───┴───┘
+*/
+#define LAYOUT_60_iso_5x1u_split( \
```
Comment (zvecr) [inline keyboards/dz60/dz60.h]: This Layout macro is not actually adding a split spacebar which is what the `_split` part usually designates.

File: keyboards/dz60/keymaps/iso_de_root/README.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# ISO DE layout
```
Comment (zvecr) [inline keyboards/dz60/keymaps/iso_de_root/README.md]: Can this file be renamed to lowercase

File: keyboards/dz60/dz60.h (hunk 1 above)
Comment (1r00t) [reply keyboards/dz60/dz60.h]: Okay, I will rename this to split_rshift?

Comment (noroadsleft) [reply keyboards/dz60/dz60.h]: `LAYOUT_60_iso_5x1u_split_rshift` would be consistent with our layout naming scheme.

**Comment by drashna** [comment]
Thanks!

**Pull request merged**